front/pages/w/[wId]/builder/assistants/new.tsx

```diff
@@ -1,27 +1,47 @@
-import { DataSourceType } from "@dust-tt/types";
-import { UserType, WorkspaceType } from "@dust-tt/types";
-import { AppType } from "@dust-tt/types";
-import { PlanType, SubscriptionType } from "@dust-tt/types";
+import {
+  AgentConfigurationType,
+  AppType,
+  DataSourceType,
+  isDatabaseQueryConfiguration,
+  isDustAppRunConfiguration,
+  isRetrievalConfiguration,
+  PlanType,
+  SubscriptionType,
+  UserType,
+  WorkspaceType,
+} from "@dust-tt/types";
 import { GetServerSideProps, InferGetServerSidePropsType } from "next";
 
 import AssistantBuilder, {
+  AssistantBuilderInitialState,
   BUILDER_FLOWS,
   BuilderFlow,
 } from "@app/components/assistant_builder/AssistantBuilder";
 import { getApps } from "@app/lib/api/app";
+import { getAgentConfiguration } from "@app/lib/api/assistant/configuration";
 import { getDataSources } from "@app/lib/api/data_sources";
 import { Authenticator, getSession, getUserFromSession } from "@app/lib/auth";
 
+import { buildInitialState } from "./[aId]";
```

Comment:
let's make it @app based?

Reply:
For this case I find it cleaner to have a relative import. They are tight together but creating an extra file just for that is a bit too much. I am ok with moving it, but just for the rational.